homeassistant/components/weatherflow_cloud/const.py

```diff
@@ -7,3 +7,25 @@
 
 ATTR_ATTRIBUTION = "Weather data delivered by WeatherFlow/Tempest REST Api"
 MANUFACTURER = "WeatherFlow"
+
+ICON_MAP = {
```

Comment:
It might be a bit clearer to refer to this as a state map.

If I understand correctly, this is mapping states from the integration to states within HA, and the actual mapping to icons happens later with `icons.json`.

Initially I was confused because I was looking for these strings in the MDI library.

If you agree, the same change might apply to the docs PR.

Reply:
What do you suggest for a name?

Reply:
Done! :)